### DISCUSSION TITLE
Create new note from picker

### DISCUSSION THREAD

**Discussion by mhanberg** [zk-org/zk-nvim#200] [Ideas]
### Verify

- [X] I searched the existing discussions for similar ideas

### Share your idea or feature request

I maintained  my own ZK plugin for nvim for a while ([mhanberg/zk.nvim](https://github.com/mhanberg/zk.nvim)) (before zk-nvim) existed), but recently switched over to zk-nvim to reduce my own maintenance burden.

One missing feature I'm noticing is the ability to create a new note from the find notes picker with the query you've typed in.

In fzf-lua/fzf at least, this takes the form of hitting `ctrl-e` in the fzf window to create a new note with that query. This is pretty common in the "notational velocity" style of note taking.

## Feature

- Hit `ctrl-e` to create a new note from the query in the picker
- OR: allow a callback/action to be passed to include other actions

## Notes

- Code in mhanberg/zk.nvim for this [feature](https://github.com/mhanberg/zk.nvim/blob/5fceabf35ec834ca5602819248259d116669b663/lua/zk/notes/init.lua#L88-L91)

**Comment by tjex** [comment]
This is a pretty nice idea. I'd accept a PR for that and help work on it.
And your plugin looks great. Sorry to hear that you're moving on for it, but great to have you over here :)

**Reply by mhanberg** [reply]
> And your plugin looks great. Sorry to hear that you're moving on for it, but great to have you over here :)

Thanks! zk-nvim seems more robust and has more features, so it was time to switch over.

> This is a pretty nice idea. I'd accept a PR for that and help work on it.

I might find some time, but am rather busy after hours these days 😩.
